project/dexter-vs/Dexter/PeerReview/Core/PReviewConstants.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dexter.PeerReview
+{
+    public static class PReviewConstants
+    {
+        public static readonly string COMMENT_OWNER = "ICommentOwner";
```

Comment:
It is referenced below 2 places.
* Dexter.PeerReview.Core\PeerReviewMargin.cs - (53, 84) : textView.TextBuffer.Properties.GetProperty(PeerReviewConstants.COMMENT_OWNER);
* Dexter.PeerReview.Core\PeerReviewTagger.cs - (59, 72) : this.textBuffer.Properties.AddProperty(PeerReviewConstants.COMMENT_OWNER, this);

It is to bind the peer-review comments to the text buffer's property.